pyface/data_view/i_data_view_widget.py

```diff
@@ -228,9 +228,7 @@ def _create(self):
         This method should create the control and assign it to the
         :py:attr:``control`` trait.
         """
-        self.control = self._create_control(self.parent)
-        self._initialize_control()
-        self._add_event_listeners()
+        super()._create()
```

Comment:
~does this PR depend on #1013 ?~ It looks like this PR makes some of the changes that are included in #1013. Is that intentional? Do the changes in this PR depend on #1013 ?

Reply:
No, it doesn't depend on #1013 because it adds a new class between these classes and the base `Widget` class which _doesn't_ call `super`.

Now that #1013 is merged we will want to add that `super` call, but it can safely be in another PR if needs be (or if I make any modifications to this I will add it in to this PR).

Reply:
I've merged in main and resolved all of this.